src/Snackbar/index.scss

```diff
@@ -0,0 +1,41 @@
+.nds-snackbar {
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding: var(--space-m);
+  gap: var(--space-m);
+  min-height: rem(64px);
+  border-radius: rem(24px);
+  box-shadow: var(--elevation-high);
+  width: max-content;
+  min-width: 330px;
+  max-width: 80vw;
+  margin: 0 auto;
+
+  @include atMediaUp("m") {
+    flex-wrap: nowrap;
+    border-radius: rem(56px);
+    padding: var(--space-s) var(--space-m);
+  }
```

Comment:
The "default" properties target _all_ viewport sizes, making mobile sizes essentially the default in our mobile-first media queries.

This `atMediaUp` generates a media query that targets "m" size viewports _and larger_.
Setting `nowrap` on the flex forces all children of Snackbar into one line.